internal/inbox/inbox.go

```
@@ -430,11 +505,39 @@ func (m *Manager) Update(id int, inbox imodels.Inbox) (imodels.Inbox, error) {
+			return imodels.Inbox{}, m.persistenceError("fetching inbox alias verification state", err)
+		}
+		for _, state := range states {
+			verificationStates[strings.ToLower(state.Email)] = state
+		}
+	}
+	if err := tx.Stmtx(m.queries.Update).Get(&updatedInbox, id, inbox.Channel, encryptedConfig, inbox.Name, inbox.From, inbox.CSATEnabled, inbox.PromptTagsOnReply, inbox.Enabled, inbox.Secret, inbox.LinkedEmailInboxID, inbox.FromNameTemplate); err != nil {
 		m.lo.Error("error updating inbox", "error", err)
-		return imodels.Inbox{}, envelope.NewError(envelope.GeneralError, m.i18n.T("globals.messages.somethingWentWrong"), nil)
+		return imodels.Inbox{}, m.persistenceError("updating inbox", err)
+	}
+	if _, err := tx.Stmtx(m.queries.DeleteAddresses).Exec(id); err != nil {
```

golangci-lint (2.13.2)

[error] (*database/sql.Stmt).Exec must not be called. use (*database/sql.Conn).ExecContext

(noctx)